pages/operations/cluster-configuration.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Cluster Configuration
+description: DevOps workflows that involve editing your cluster Terraform.
+---
+
+Plural offers a set of sane defaults to spin up a one-size-fits-all Kubernetes cluster, but there will be cases
```

Comment:
this is a really good paragraph. that's all.

Reply:
aw thanks :)